client/components/sticky-panel/style.scss

```diff
@@ -11,4 +11,4 @@
 	.sticky-panel.is-sticky .sticky-panel__content {
 		margin-top: 8px;
 	}
-}
\ No newline at end of file
+}
```

Comment:
I know this is just fixing a missing newline that all files should have (and was probably done automatically by your editor) but we should probably drop this change to keep all changes in this PR in the woocommerce directory.